Lesson7_OOPS.cpp

```diff
@@ -0,0 +1,106 @@
+/* DESIGN A VENDING MACHINE */
+#include<iostream>
+using namespace std;
+class Inventory
+{
+	int coke,sprite,chips;
```

Comment:
I know this is just an example, but you would probably want these to be added at run time, so that you could change your inventory without changing the underlying code.

Reply:
Do these stand for prices?  That's not very obvious from the name.  Also, is price always an int?  Could something cost $1.25?